tools/format/examples/vehicle_part.json

```diff
@@ -14,14 +14,15 @@
     "breaks_into": [
       { "item": "steel_lump", "count": [ 3, 6 ] },
       { "item": "steel_chunk", "count": [ 3, 6 ] },
-      { "item": "scrap", "count": [ 3, 6 ] }
+      { "item": "scrap", "min": 3, "max": 6 }
```

Comment:
No worries. Now that you have the hang of I'd encourage you to submit more linter rules so we can extend coverage to most of the core JSON